### PR TITLE
Created jobs endpoint

### PR DESCRIPTION
**Pull request by Manjukcthapa**
# Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Created jobs endpoints and able to get the job data from third parties API.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ *] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] There are no merge conflicts

### CODE REVIEW

File: routes/city.js [hunk 1]
```diff
@@ -542,4 +537,36 @@ router.post("/spec-ds", tokenAuthentication, async (req, res) => {
   }
 });
 
+router.get("/jobs", async (req, res) => {
+  const input = req.query;
+  async function getJobs(params) {
```
Comment (shweps13) [inline routes/city.js]: Remember the main backend rule - never trust the user!
You need to add logical iterations where function will check that request not empty - like if (!input) { say something to the user, he forgot the query}

Comment (shweps13) [reply routes/city.js]: also will be better to check input thru RegExp - there is should be the name of the city without special characters

File: routes/city.js [hunk 2]
```diff
@@ -542,4 +537,36 @@ router.post("/spec-ds", tokenAuthentication, async (req, res) => {
...
+    try {
+      const response = await axios.get(
+        `https://indreed.herokuapp.com/api/jobs`, {
+          params
```
Comment (shweps13) [inline routes/city.js]: Is that possible to filter it and get just tech jobs? Or maybe there are API parameters?
Our project has tech direction

**Review by shweps13 — CHANGES_REQUESTED** [review]
Check input  please

File: routes/city.js (hunk 2 above)
Comment (Manjukcthapa) [reply routes/city.js]: Yes it is. You can use q=Software for Software Jobs, or whatever you need

File: routes/city.js (hunk 1 above)
Comment (Manjukcthapa) [reply routes/city.js]: okey, I understand and fixed it

**Pull request merged**